app/grants/views.py

```diff
@@ -2474,6 +2474,30 @@ def quickstart(request):
     }
     return TemplateResponse(request, 'grants/quickstart.html', params)
 
+def hall_of_fame(request):
+    """Display the hall of fame."""
+    hall_of_fame_query = GrantHallOfFame.objects.filter(is_published=True)
+    try:
+        hall_of_fame = hall_of_fame_query[:1][0]
+    except IndexError:
+        raise Http404
+
+    params = {
+        'active': 'hall_of_fame',
+        'title': _('Hall of Fame'),
+        'avatar_url': request.build_absolute_uri(static('v2/images/twitter_cards/grants10.png')),
+
+        'total_donations': hall_of_fame.total_donations,
+        'top_individual_donors_url': hall_of_fame.top_individual_donors.url,
+        'top_matching_partners_url': hall_of_fame.top_matching_partners.url,
+        'top_individual_donors_mobile_url': hall_of_fame.top_individual_donors_mobile.url,
+        'top_matching_partners_mobile_url': hall_of_fame.top_matching_partners_mobile.url,
+        'graduated_grantees_description': hall_of_fame.graduated_grantees_description,
+        'share_your_story_email': hall_of_fame.share_your_story_email,
+        'graduated_grantees': hall_of_fame.get_grantees_data(),
+    }
+
+    return TemplateResponse(request, 'grants/hall_of_fame.html', params)
```

Comment:
^ Hmm so we are trying to move away from using django templating from new pages!
Let's instead throw a new API endpoint for just the response
https://github.com/gitcoinco/web/pull/9377/files

^ Use that as an example

Reply:
I can make this change if required.
Are there any objections  @octavioamu / @chibie ?